app/assets/javascripts/admin/components/admin-report-table.js.es6

```diff
@@ -79,8 +79,8 @@ export default Ember.Component.extend({
     if (sortLabel) {
       const compare = (label, direction) => {
         return (a, b) => {
-          let aValue = label.compute(a).value;
-          let bValue = label.compute(b).value;
+          let aValue = label.compute(a, { useSortProperty: true }).value;
+          let bValue = label.compute(b, { useSortProperty: true }).value;
```

Comment:
```suggestion
           const bValue = label.compute(b, { useSortProperty: true }).value;
```

Reply:
Nice! I believe we should also change `aValue` to be `const`.